resources/js/beatmap-discussions/nominations.tsx

```diff
@@ -634,12 +634,36 @@ export class Nominations extends React.PureComponent<Props> {
             : formatDate(rankingEta)
           : trans('beatmaps.nominations.rank_estimate.soon');
 
+        const unresolvedProblems = this.props.beatmapset.discussions.filter((d) => !d.resolved && ['problem', 'suggestion'].includes(d.message_type));
```

Comment:
Use `this.props.discussionsState.discussionsByFilter.pending`

Reply:
hm, how would you suggest using this...? `this.props.currentDiscussions.byFilter.pending` is ` Record<"reviews" | "generalAll" | "general" | "timeline", Partial<Record<number, BeatmapsetDiscussionDefault>>>` so it doesn't work as nicely

Reply:
`discussionsState.discussionsByFilter.pending` is new, pull the latest changes from `master`